ras_party/support/session_decorator.py

```diff
@@ -32,6 +32,7 @@ def wrapper(*args, **kwargs):
             raise
         except Exception as e:
             logger.info("Rolling-back database session.")
```

Comment:
I don't think we should be logging at info level if we raise an error at all, it's misleading. Lets remove this line and incorporate it into the exception logging

Reply:
I agree but Rhi mentioned @davidmort wanted both in?

Reply:
not quite - what i wanted was the error to be logged out as it's just currently returned in the api response https://github.com/ONSdigital/ras-party/blob/f5d645226ba4803d9509263517746208e4b0e731/ras_party/error_handlers.py. adding the two log lines into one as suggested sounds good, but really we should fix the error_handlers.py to log out errors/exceptions correctly

Reply:
Yeah for that we'll need to remove that generic RasError

Reply:
Spoken to @rhiGriff and she's got some other cards she's working on so I can help out with this